dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherOptions.nlsprops

```diff
@@ -158,11 +165,21 @@ option-desc.clean=\
 #------------------------------\n at 72 chars -- leading tab-----------\n\#
 option-key.archive=\
 \ \ \ \ --archive="path to the target archive file"
-option-desc.archive=\
-\tSpecify the archive target to be generated by the package or dump    \n\
-\taction. The target can be specified either as an absolute path or    \n\
-\tas a relative path. If this option is omitted, the archive file will\n\
-\tbe created in the server output directory. The target file name      \n\
+option-desc.dump.archive=\
+\tSpecify the archive target to be generated by the dump action.       \n\
+\tThe target can be specified either as an absolute path or as a       \n\
+\trelative path. If this option is omitted, the archive file will be   \n\
+\tcreated in the server output directory. The target file name         \n\
+\textension might influence the format of the generated archive.       \n\
+\tThe default archive format for the package action is "pax" on z/OS   \n\
+\tand "zip" on all other platforms.                                    \n\
+\tArchive format "jar" will produce a self-extracting jar similar to   \n\
+\tthe original installer archive.                                      
+option-desc.package.archive=\
+\tSpecify the archive target to be generated by the package action.    \n\
+\tThe target can be specified either as an absolute path or as a       \n\
+\trelative path. If this option is omitted, the archive file will be   \n\
+\tcreated in the server output directory. The target file name         \n\
```

Comment:
+\trelative path. If this option is omitted, the archive file **is**   \n\

Reply:
I don't think that is correct. If I use the word "is" rather than will be it would imply the option was telling the command line where to find a file, rather than where to produce a file to.